src/main/java/Objects/NetworkDevices/NetworkDevice.java

```diff
@@ -1,16 +1,25 @@
 package Objects.NetworkDevices;
 
 import java.util.ArrayList;
+import java.util.List;
 
 /**
- * Parent class for all physical network objects
+ * Parent class for all physical network devices.
  */
 public class NetworkDevice {
 
     private String id;
     private String team;
     private ArrayList<Integer> connections;
+    private int speed;
+    private Boolean hidden;
+    private int max_packet;
```

Comment:
not a huge deal but as I understand it camel-case is standard for java, so this would be maxPacket

Reply:
Matter of style opinion.
Project for class.
Not a Java developer making a professional project.
Resolving.

Reply:
its to try and keep uniformity of code throughout the project so whoever wants to look at it can easily understand it